### PR TITLE
Fixed netci.groove scripts to make CentOS and Ubuntu R2R jitstressregs builds worked.

### PR DESCRIPTION
**Pull request by rartemev**
Change for #10094. Fixes the windows test build job name for R2R JitStressRegs jobs.

### CODE REVIEW

**Comment by rartemev** [comment]
@dotnet-bot test Ubuntu jitstress2_jitstressregs0x1000

**Comment by briansull** [comment]
I want too see if this task is expected to fail....

@dotnet-bot test Windows_NT Release r2r

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT tailcallstress
@dotnet-bot test Windows_NT x86 Checked tailcallstress
@dotnet-bot test Windows_NT x64 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x10
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x80
@dotnet-bot test Windows_NT jitsse2only
@dotnet-bot test Windows_NT x86 Checked jitsse2only

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT Release r2r
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT tailcallstress
@dotnet-bot test Windows_NT x86 Checked tailcallstress
@dotnet-bot test Windows_NT x64 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x10
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x80
@dotnet-bot test Windows_NT jitsse2only
@dotnet-bot test Windows_NT x86 Checked jitsse2only

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT Release r2r
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT tailcallstress
@dotnet-bot test Windows_NT x86 Checked tailcallstress
@dotnet-bot test Windows_NT x64 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x10
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x80
@dotnet-bot test Windows_NT jitsse2only
@dotnet-bot test Windows_NT x86 Checked jitsse2only

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT x64 Checked jitstressregs0x10 R2R Build & Test

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT Release r2r
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT tailcallstress
@dotnet-bot test Windows_NT x86 Checked tailcallstress
@dotnet-bot test Windows_NT x64 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x10
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x80
@dotnet-bot test Windows_NT jitsse2only
@dotnet-bot test Windows_NT x86 Checked jitsse2only

**Comment by rartemev** [comment]
@dotnet-bot test Windows_NT Release r2r
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT tailcallstress
@dotnet-bot test Windows_NT x86 Checked tailcallstress
@dotnet-bot test Windows_NT x64 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 corefx_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstress2_jitstressregs0x1000
@dotnet-bot test Windows_NT x86 Checked jitstressregs0x1000
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x10
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x80
@dotnet-bot test Windows_NT jitsse2only
@dotnet-bot test Windows_NT x86 Checked jitsse2only

**Comment by rartemev** [comment]
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x10
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x80
@dotnet-bot test OSX Checked r2r_jitstressregs0x10
@dotnet-bot test OSX Checked r2r_jitstressregs0x80
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x10
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x80
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x10
@dotnet-bot test Windows_NT Checked r2r_jitstressregs0x80

**Comment by rartemev** [comment]
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x10
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x80
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x10
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x80

**Comment by rartemev** [comment]
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x10
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x80
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x10
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x80

**Comment by rartemev** [comment]
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x10
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x80
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x10
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x80

**Comment by rartemev** [comment]
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x10
@dotnet-bot test CentOS7.1 Checked r2r_jitstressregs0x80
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x10
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x80

**Comment by rartemev** [comment]
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x10
@dotnet-bot test Ubuntu Checked r2r_jitstressregs0x80

**Comment by rartemev** [comment]
@dotnet-bot  test ci please

File: netci.groovy [hunk 1]
```diff
@@ -2165,7 +2165,7 @@ combinedScenarios.each { scenario ->
                     if (testBuildScenario == 'coverage' || testBuildScenario == 'pri1r2r'|| testBuildScenario == 'gcstress15_pri1r2r') {
                         testBuildScenario = 'pri1'
                     }
-                    else if ( testBuildScenario == 'r2r' || isLongGc(testBuildScenario)) {
+                    else if ( testBuildScenario == 'r2r' || Constants.jitStressModeScenarios.containsKey(testBuildScenario) || isLongGc(testBuildScenario)) {
```
Comment (mmitche) [inline netci.groovy]: The r2r stress modes are in a different list.  This should be: Constants.r2rJitStressScenarios.indexOf(testBuildScenario) != -1

**Comment by rartemev** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
LGTM

**Comment by rartemev** [comment]
@dotnet-bot test OSX10.12 x64 Checked Build and Test

**Pull request merged**